ui/src/pages/Other.svelte

```diff
@@ -62,6 +62,11 @@
     if (!$state.social) return;
     item = getItem(keyStr);
     if (!item) return;
+
+    // This is a little confusing but we're merging the global list of comments
```

Comment:
move this comment over from feedpostform for posterity

Reply:
why don't we just use the same function and factor it out?